Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -29,6 +29,11 @@ then
       ruby_URL2="https://github.com/Homebrew/homebrew-portable-ruby/releases/download/2.6.3/portable-ruby--2.6.3.x86_64_linux.bottle.tar.gz"
       ruby_SHA="e8c9b6d3dc5f40844e07b4b694897b8b7cb5a7dab1013b3b8712a22868f98c98"
       ;;
+    aarch64)
+      ruby_URL="$HOMEBREW_BOTTLE_DOMAIN/bottles-portable-ruby/portable-ruby--2.6.3.aarch64_linux.bottle.tar.gz"
```

Comment:
Can we get a documented (and ideally CId) Docker or VM installation method that doesn't require native hardware before we re-add these?

Reply:
For how often we update `portable-ruby`, I'm not convinced that it's worth the effort. That said, I'll look into it, and try to determine whether it's easy or hard to use Docker on `x86_64` hardware to emulate ARM hardware. This bottle was built using @zbeekman 's AArch64 hardware.

Reply:
There is cloud arm support from Drone... I've been meaning to tinker with this for a while for work. https://blog.drone.io/drone-cloud/

Also, I may be able to get a more robust distro for my RK3399 that would allow me to run docker on the RK3399 machine, or put my rpi 4 in my office and launch a DIY CI setup. (It's the same grid as the white house, I've never had a power outage in the 10 years that I've lived in this neighborhood. My office has a "business class" connection with slightly better up B/W than my home network.)

Reply:
Looks like it should be possible to emulate ARM hardware with Docker on x86_64!

https://docs.docker.com/docker-for-mac/multi-arch/
https://docs.docker.com/buildx/working-with-buildx/
https://engineering.docker.com/2019/04/multi-arch-images/

Reply:
FYI, you can run docker with ARM containers directly with Docker Desktop for Mac or Docker Desktop for Windows. They have built-in support for running cross architecture containers through qemu.

Alternatively, you can install qemu on Linux. There are several existing tools to do this:
* https://github.com/multiarch/qemu-user-static
* https://github.com/hypriot/qemu-register

The only downside is that the performance may not be as good as native hardware. But it should be pretty easy to automate the whole workflow in CI.